libs/vkd3d/meta.c

```diff
@@ -620,6 +620,91 @@ void vkd3d_copy_image_ops_cleanup(struct vkd3d_copy_image_ops *meta_copy_image_o
     vkd3d_free(meta_copy_image_ops->pipelines);
 }
 
+static VkResult vkd3d_meta_create_swapchain_render_pass(struct d3d12_device *device,
+        VkFormat format, VkRenderPass *render_pass)
+{
+    const struct vkd3d_vk_device_procs *vk_procs = &device->vk_procs;
+    VkRenderPassCreateInfo render_pass_info;
+    VkAttachmentDescription attachment_desc;
+    VkAttachmentReference attachment_ref;
+    VkSubpassDescription subpass_desc;
+    VkSubpassDependency subpass_dep;
+
+    render_pass_info.sType = VK_STRUCTURE_TYPE_RENDER_PASS_CREATE_INFO;
+    render_pass_info.pNext = NULL;
+    render_pass_info.flags = 0;
+    render_pass_info.attachmentCount = 1;
+    render_pass_info.pAttachments = &attachment_desc;
+    render_pass_info.subpassCount = 1;
+    render_pass_info.pSubpasses = &subpass_desc;
+    render_pass_info.dependencyCount = 1;
+    render_pass_info.pDependencies = &subpass_dep;
+
+    attachment_desc.loadOp = VK_ATTACHMENT_LOAD_OP_CLEAR;
```

Comment:
Do we really need to clear? DXVK uses `DONT_CARE` here.

Reply:
Right now scaling is not implemented properly, I will fix it later.

Maybe conditionally use DONT_CARE depending on whether we are NONE or STRETCH.
-> note: STRETCH and ASPECT_RATIO_STRETCH are the same for Win32 swapchains.

Reply:
Added a load_op member in the key.